corehq/apps/app_manager/decorators.py

```diff
@@ -146,5 +147,39 @@ def _no_conflict(*args, **kwargs):
     return _no_conflict
 
 
+def avoid_parallel_build_request(fn):
+    @wraps(fn)
+    def new_fn(app, comment, user_id, *args, **kwargs):
```

Comment:
I think you should just make this signature `*args, **kwargs` so that you don't need to change it if the signature of the wrapped function changes.

Reply:
and then use
```
app = args[0]
```
?

Reply:
Isn't that a little delicate. If the signature changes, this will more explicitly show that the change needs to be done?

Reply:
ah, I see. Ignore my comment, I didn't look closely enough at this function.